css/main.css

```diff
@@ -133,7 +133,7 @@ dt {
     padding: 40px;
 }
 
-.row {
+.row.row-index {
```

Comment:
I'm fairly sure there should be a space between those

Reply:
I put a space between and it did not work. it needs to target a div that has row and row-index.

Reply:
oh duh... i read it wrong... somehow my brain put those in different elements... [face_palm]